frontend/VolunteerOne/components/Modals/NewPostModal.js

```diff
@@ -0,0 +1,208 @@
+// This file was added by Matt
+// It contains the component for a New post modal item 
+// which is dispayed when a user wants to make a new announcement
+// it works like a popup. 
+
+import React from 'react';
+import {Alert, Modal, StyleSheet, Pressable, View, Dimensions, TextInput, Image, ScrollView} from 'react-native';
+import { Block, Text, theme } from 'galio-framework';
+import { argonTheme } from "../../constants";
+import ImagePicker from './ImagePicker.js';
+
+
+const { width, height } = Dimensions.get("screen");
+
+function handleCreatePostBtnClick() {
+// write function for create announcement button here
+
```

Comment:
(Same as below) We can potentially add console.log() messages here as a placeholder here to make sure we're actually reaching this method on click